src/Button/GeoLocationButton/GeoLocationButton.jsx

```diff
@@ -0,0 +1,265 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import OlMap from 'ol/map';
+import OlGeolocation from 'ol/geolocation';
+import OlGeomLineString from 'ol/geom/linestring';
+import OlOverlay from 'ol/overlay';
+
+import ToggleButton from '../ToggleButton/ToggleButton.jsx';
+import { MathUtil } from '../../index';
+
+import './GeoLocationButton.less';
+import mapMarker from '../../../assets/geolocation-marker.png';
+import mapMarkerHeading from '../../../assets/geolocation-marker-heading.png';
```

Comment:
I'd prefer to store the assets of a component within its component folder to increase the cohesion. Any thoughts?